validate.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const index = require('./index');
+const commander = require('commander');
```

Comment:
[template main](https://github.com/screwdriver-cd/template-main/blob/1d044d343dc0de28f25b3b3018972a012e119181/validate.js#L6) uses `nomnom` but it is unmaintained. So I use `commander` instead of `nomnom`.
https://github.com/harthur/nomnom